src/views/monitoring/alert/modules/EscalationRulesInputForm.vue

```diff
@@ -22,7 +22,19 @@
                 <p-badge outline style-type="gray">{{ idx + 1 }}</p-badge>
             </span>
             <span class="col-notification">
-                <p-select-dropdown v-model="rule.notification_level" :items="NOTIFICATION_LEVELS" />
+                <p-select-dropdown v-model="rule.notification_level" :items="NOTIFICATION_LEVELS">
+                    <template #menu-item--format="{item}">
+                        <p-radio v-model="rule.notification_level"
+                                 :selected="rule.notification_level"
```

Comment:
엇 !  `selected` 와 `v-model` 이 같이 있어영!
```suggestion
                        <p-radio v-model="rule.notification_level"
```

Reply:
감사요 반영했습니다!